index.js

```diff
@@ -1,7 +1,59 @@
 // Iteration 1: Names and Input
+let hacker1 = "Ivan";
+let hacker2 = "Jose";
```

Comment:
```suggestion
const hacker1 = "Ivan";
const hacker2 = "Jose";
```
Salvo que los valores vayan a ser reasignados, siempre siempre siempre :D, es mejor poner const, que el let los carga el diablo